docs/Search.md

```diff
@@ -0,0 +1,27 @@
+# Search
+
+Chatterino has Search Popup which lets you quickly find messages in the chat. You can open it with the `Ctrl+F` shortcut.
```

Comment:
```suggestion
Chatterino has a Search Popup which lets you quickly find messages in the chat. You can open it with the `Ctrl+F` shortcut or through the split menu.
```

Not sure If I'm right about the naming of the split menu but you can change it.

Reply:
I've been considering adding this, but I wanted to keep it short and simple to not overwhelm users with the amount of text there. I know I provided only 1 option out of 2 that are available, but honestly i feel like just being able to Ctrl+F the popup is enough to know about.